### ISSUE TITLE
No screenshot option in context menu when text has been selected

### ISSUE THREAD

**Issue by mikeconley** [mozilla-services/screenshots#5330]
Originally filed as [bug 1520298](https://bugzilla.mozilla.org/show_bug.cgi?id=1520298):

> User Agent: Mozilla/5.0 (X11; Linux x86_64; rv:64.0) Gecko/20100101 Firefox/64.0
> 
> Steps to reproduce:
> 
> It's common practice with screenshots to select some text with the mouse, as a means to highlight the text. For instance: https://twitter.com/martinkrafft/status/1084884562226733056
> 
> However, once text is selected, the right-mouse-button menu of Firefox no longer has the option to take a screen shot. It's still possible to go via the page actions, but it would be nice if the context menu also had the option.
> 
> Actual results:
> 
> Once a text selection has been made, the context menu no longer shows the screenshot option.
> 
> Expected results:
> 
> For consistency, the screenshot option should be available, even when text has been selected, and there is no real reason why that shouldn't be possible.

**Comment by madduck** [comment]
You are awesome, @mikeconley . Thank you!

**Comment by SoftVision-CosminMuntean** [comment]
I have verified this functionality using the latest Screenshots (37.0.0) dev version, but the "Take a Screenshot" option is still not available in the context menu if a text is selected.
Tested on Windows 7 x64, Mac 10.14 and Arch Linux.
Here is a screenshot with the context menu:
![scr1](https://user-images.githubusercontent.com/19226534/51972860-0f73ac00-2485-11e9-829f-27550b399e98.png)


@ianb @6a68 is there any chance that the change didn't land in the latest Screenshots dev version installed from [here](https://screenshots.dev.mozaws.net/homepage/install-test-local.html)?
